config/recipes/beats/metricbeat_openshift.yaml

```diff
@@ -0,0 +1,244 @@
+apiVersion: beat.k8s.elastic.co/v1beta1
```

Comment:
Could you add description of that file to `README.md` and `docs/orchestrating-elastic-stack-applications/beat.asciidoc`? This way it's accessible both through the repo and at https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-beat-configuration-examples.html.

Reply:
Added the information to the `README.md`, I will soon add an update to the docs incdluding the openshift example.

Reply:
now added to the [docs](https://github.com/elastic/cloud-on-k8s/blob/openshift-monitoring-beats-recipe/docs/orchestrating-elastic-stack-applications/beat.asciidoc#openshift-monitoring)